client/src/main/java/net/spals/oembed4j/client/cache/OEmbedResponseCache.java

```diff
@@ -1,78 +0,0 @@
-package net.spals.oembed4j.client.cache;
-
-import com.google.common.annotations.VisibleForTesting;
-import net.jodah.expiringmap.ExpirationPolicy;
-import net.jodah.expiringmap.ExpiringEntryLoader;
-import net.jodah.expiringmap.ExpiringMap;
-import net.jodah.expiringmap.ExpiringValue;
-import net.spals.oembed4j.client.OEmbedClient;
-import net.spals.oembed4j.model.OEmbedRequest;
-import net.spals.oembed4j.model.OEmbedResponse;
-
-import java.net.URI;
-import java.util.Optional;
-import java.util.concurrent.TimeUnit;
-
-/**
- * A cache to store {@link OEmbedResponse}s based
- * on their cache_age property.
- *
- * @author tkral
- */
-public class OEmbedResponseCache {
```

Comment:
Removing the cache class entirely. The logic was too complex as a separate class.